vue/components/ui/atoms/reaction/reaction.vue

```diff
@@ -60,6 +61,11 @@
     width: 15px;
 }
 
+.reaction > .icon {
+    height: 15px;

```

Comment:
How was this image being resized previously when using the `<img>`... there was no CSS?